config/config.go

```diff
@@ -24,6 +24,7 @@ type InsertConfig struct {
 	SleepSeconds           int    `mapstructure:"sleep_seconds"`
 	DataDir                string `mapstructure:"data"`
 	FreeSpaceRequiredBytes int64  `mapstructure:"free_space_required_bytes"`
+	NumberOfWorkers        int    `mapstructure:"no_of_workers"`
```

Comment:
Hi! I made a mistake in my original comment. This should be in the `IngestConfig` struct on line 30. I apologize! Maybe this could be named `S3UploadWorkers` or something more descriptive?